src/node_modules.cc

```diff
@@ -332,42 +334,32 @@ void BindingData::GetNearestParentPackageJSON(
     path_value_str.push_back(std::filesystem::path::preferred_separator);
   }
 
-  auto package_json =
-      TraverseParent(realm, std::filesystem::path(path_value_str));
+  args.GetReturnValue().Set(path_value_str);
+}
+
+void BindingData::GetNearestParentPackageJSON(
+  const v8::FunctionCallbackInfo<v8::Value>& args
+) {
+  std::string path_value_str = FindNearestParentPackageJSON(args);
```

Comment:
No need to append the type to the variable name 

```suggestion
  auto path = FindNearestParentPackageJSON(args);
```